src/pages/Entries/Entries.tsx

```diff
@@ -15,26 +18,72 @@ export default function Entries() {
   const [editing, setEditing] = useState(false);
   const prevSelectedEntryId = useRef<number | undefined>(undefined);
   const { user } = useUser();
+  const { api } = useApi();
+  const queryClient = useQueryClient();
+  const numStillMutating = useIsMutating({ mutationKey: MutationKeys.MUTATE_HAPPINESS });
+  const [networkingState, setNetworkingState] = useState(
+    Constants.LOADING_MUTATION_TEXT.toString(),
+  );
+
+  const updateHappinessTimeout = useRef<number | undefined>(undefined);
+  const updateHappiness = () => {
+    //@ts-ignore TODO get rid of user_id field once backend is actually updated
+    const { id, author, user_id, ...happinessToPost } = selectedEntry;
```

Comment:
the backend is actually updated right?

Reply:
it's not, check [the docs](https://happiness-app-backend.herokuapp.com/docs#/operations/happiness_get_happiness_time)

Reply:
should I just create a dev server lol (i might know how to make one for free using a different platform, they would share the prod db though)

Reply:
it would be great if you could yes